### DISCUSSION TITLE
Getting error after enabling MFA - DescribeInstances operation: AWS was not able to validate the provided access credentials

### DISCUSSION THREAD

**Discussion by ashu27483** [cloud-custodian/cloud-custodian#7422] [General]
### Ask your question

I am trying to get custodian to connect to a EC2 instance created by normal user for which Virtual MFA is enabled ( Google Auth), but now I am unable to run any Cloud Custodian script even for EC2 instance created by AWS root user. ( Earlier same script was working fine)

botocore.exceptions.ClientError: An error occurred (AuthFailure) when calling the DescribeInstances operation: AWS was not able to validate the provided access credentials
2021-10-01 00:15:59,784: custodian.commands:ERROR The following policies had errors while executing
 - my-first-policy1

Additionally, is there any way to install Cloud Custodian on Windows 10, tried the instruction as per doc, but it didnt work.

Thanks.

**Comment by kapilt** [comment]
this is really independent of custodian, you should try to get the aws cli working independently of custodian first to validate you have valid cloud credentials
